src/vpp_plugins/vpp-chaperone/VppConfigManager.cpp

```diff
@@ -792,6 +798,56 @@ void VppConfigManager::doCpeConfig (VppClient &vppClient)
               doCpePolicerConfig (vppClient, kv.first.asString (),
                                   kv.second["policers"]);
             }
+          else
+            {
+              vppClient.getClassifierTableIds (tableIds);
```

Comment:
The code itself looks good. Can you put this in a separate function for the sake of modularity/cleanliness? e.g. `deleteExistingPolicers` or something similar

Reply:
Addressed Review Comments